lib/matplotlib/axes/_axes.py

```diff
@@ -3514,7 +3514,7 @@ def boxplot(self, x, notch=None, sym=None, vert=None, whis=None,
             fliers default to 'b+'  If you want more control use the
             flierprops kwarg.
 
-        vert : bool, optional (True)
+        vert : bool, optional, default: True
             If `True` (default), makes the boxes vertical. If `False`,
```

Comment:
doesn't need the  default in `True`(default) anymore

Reply:
done (and reworded to be consistent with violinplot)

Reply:
Holds also for a number of other cases in the PR, but we have to clean that up globally anyway.